### PR TITLE
UFS: fix which ufs we use.

### PR DESCRIPTION
**Pull request by rminnich**
It's time to move to the one in Harvey-OS.

This is tested and working for supporting harvey guests.

I have a feeling I have the commands not quite right, but OTOH
they work.

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -4,7 +4,8 @@ git submodule init
 git submodule update
 echo Building the build tool...
 GOBIN="$(pwd)/util" GOPATH="$(pwd)/util/third_party:$(pwd)/util" go get -d harvey/cmd/... # should really vendor these bits
-GOBIN="$(pwd)/util" GOPATH="$(pwd)/util/third_party:$(pwd)/util" go install github.com/rminnich/ninep/srv/examples/ufs harvey/cmd/...
+GOBIN="$(pwd)/util" GOPATH="$(pwd)/util/third_party:$(pwd)/util" go get github.com/Harvey-OS/ninep/cmd/ufs harvey/cmd/...
+GOBIN="$(pwd)/util" GOPATH="$(pwd)/util/third_party:$(pwd)/util" go install github.com/Harvey-OS/ninep/cmd/ufs harvey/cmd/...
```
Comment (sevki) [inline bootstrap.sh]: when you do go get w/o `-d` do you still need to do `go install`

**Comment by sevki** [comment]
this doesn't change the [git submodule](https://github.com/Harvey-OS/harvey/blob/master/.gitmodules) nor the [reference to it's commit ](https://github.com/Harvey-OS/harvey/tree/master/util/third_party/src/github.com/rminnich). either both should be dropped or changed. 

Word of warning: removing git submodules is not as easy as deleting those files.

**Comment by elbing** [comment]
LGTM

**Comment by elbing** [comment]
Removing submodule in another PR

**Comment by elbing** [comment]
What happens with this?

**Comment by rminnich** [comment]
I could not tell if sevki thought it was ok, I'd like his LGTM.

**Comment by sevki** [comment]
LGTM if you apply this patch https://github.com/rminnich/harvey/pull/3

**Comment by rminnich** [comment]
I don't like calling sevki's tool 'nuke' but we can fix that later.

**Pull request merged**